.devcontainer/docker-compose.yml

```diff
@@ -26,8 +26,8 @@ services:
     network_mode: service:db
 
   db:
-    #image: postgres:latest
-    image: postgres:15.4
+    image: postgres:latest
+    # image: postgres:15.4
```

Comment:
Do we need to leave this comment in?

Reply:
Actually, I think we want to use a specific postgres version and try to match it to CI/CD.  The problem with using `latest` is that one day you may suddenly get an error when you didn't change anything in your codebase due to the fact that you're suddenly working with a different version of postgres.  This definitely matters for postgres because I noticed the schema dump changed when we changed the version.